css/css-overflow/line-clamp/line-clamp-with-floats-008.html

```diff
@@ -4,10 +4,9 @@
 <link rel="author" title="Andreu Botella" href="mailto:abotella@igalia.com">
 <link rel="help" href="https://drafts.csswg.org/css-overflow-4/#line-clamp">
 <link rel="match" href="reference/line-clamp-with-floats-008-ref.html">
-<meta name="assert" content="If the line-clamp container is an independent formatting context, it will clear contained floats before the clamp point, even if they extend beyond the last line. Lines past the clamp point will remain hidden.">
+<meta name="assert" content="Line-clamp containers create an independent formatting context, so they will clear contained floats before the clamp point, even if they extend beyond the last line. Lines past the clamp point will remain hidden.">
```

Comment:
If I understand correctly, a float sets up a parallel fragmentation context from the content outside it, and the forced break in the main content caused by `max-lines: 4` should not affect the float. Since there is no height restriction on the line-clamp container, the float would not have any unforced breaks, right?

Trying a similar example with multicol, this seems to be the behavior in all browsers: https://codepen.io/andreubotella/pen/eYaeYVp. Note that Firefox only support `break-before` in print mode, so it will not show the same behavior as other browsers when rendering the page normally, but it will in print preview.

Reply:
I added a comment to both of these tests explaining this.

Reply:
@frivoal Ping

Reply:
Sorry, you are right, this is supported by https://www.w3.org/TR/css-break-3/#parallel-flows.

Note to self: It would be different for an unforced break (for example `continue: discard; max-height: 100px`), but in the case of a forced break the float is unaffected.